### DISCUSSION TITLE
.ssh not working with wsl to add new repositories.

### DISCUSSION THREAD

**Discussion by Michelkingpalmroth** [community/community#38972] [New to GitHub]
Hello! I came thus far that I can pull things from github and clone them, but I can't add new resp to my own project from the command line: all i get is: 

"remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication." 

I have a public SSH key added that i made within the command line but I get no further, and all this reading didn't make me any wiser. Do you guys know what to do?

**Comment by jfullstackdev** [comment]
maybe it's Personal Access Token 

you should go here and generate the token, the next time it's asking for password, enter it instead

https://github.com/settings/tokens

you should be logged-in when accessing the link above

**Comment by airtower-luna** [comment]
That error message implies your *repository* isn't configured with an SSH URL, and uses HTTPS. You need to either change the URL to the SSH one (you can find the correct URL on the repository page, behind the green "Code" button), or if you want to keep using HTTPS use a token as @jdevstatic mentioned.

To change the URL, use the [`git remote set-url`](https://git-scm.com/docs/git-remote#Documentation/git-remote.txt-emset-urlem) command.
